maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/IntegrationTestMojo.java

```diff
@@ -55,6 +56,13 @@
     private static final String FAILSAFE_IN_PROGRESS_CONTEXT_KEY = "failsafe-in-progress";
 
     /**
+     * The path representing project <em>jar</em> archive, if exists; Otherwise the directory containing generated
```

Comment:
This unpleasang wording happens throughout: JAR archive == Java archive archive. This is redundant. JAR file sounds better.